draft-ietf-tls-tls13.md

```diff
@@ -1582,6 +1582,13 @@ when static RSA is used.
 Note: This is an update to TLS 1.2 so in practice many TLS 1.2 clients
 and servers will not behave as specified above.
 
+RFC EDITOR: PLEASE REMOVE THE FOLLOWING PARAGRAPH
```

Comment:
Note that this won't be a following paragraph, it's markdown, so this will be part of the same paragraph :)